### PR TITLE
new package: gpsim-0.30.0

### PR DESCRIPTION
**Pull request by bobertlo**
#7769

### CODE REVIEW

File: srcpkgs/gpsim/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+license="GPL-2.0-or-later"
+homepage="full-featured software simulator for Microchip PIC mictocontrollers"
+homepage="http://gpsim.sourceforge.net/"
+distfiles="https://sourceforge.net/projects/gpsim/files/gpsim/${version}/gpsim-${version}.tar.gz"
+checksum=e1927312c37119bc26d6abf2c250072a279a9c764c49ae9d71b4ccebb8154f86
+subpackages="gpsim-devel"
+
+gpsim-devel_package() {
+	short_desc+=" - development files"
+	depends="gtk+-devel popt-devel readline-devel"
```
Comment (maxice8) [inline srcpkgs/gpsim/template]: needs to include the main package with -${version}_${revision} restriction

File: common/shlibs [hunk 2]
```diff
@@ -3399,3 +3399,10 @@ libmp3splt.so.0 libmp3splt-0.9.2_1
 libliquid.so.1 liquid-dsp-1.3.1_1
 libsnore-qt5.so.0.7 snorenotify-0.7.0_1
 libsnoresettings-qt5.so.0.7 snorenotify-0.7.0_1
+libgpsimcli.so.0 gpsim-0.30.0_1
+libgpsimgui.so.0 gpsim-0.30.0_1
+libgpsim_eXdbm.so.0 gpsim-0.30.0_1
+libgpsim.so.0 gpsim-0.30.0_1
+libgpsim_modules.so.0 gpsim-0.30.0_1
+libgpsim_extras.so.0 gpsim-0.30.0_1
+libgpsim_dspic.so.0 gpsim-0.30.0_1
```
Comment (maxice8) [inline common/shlibs]: please don't add it unless required

File: srcpkgs/gpsim/template [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	pkg_install() {
+		vmove "usr/include/gpsim/*.h"
```
Comment (maxice8) [inline srcpkgs/gpsim/template]: no need to glob the *.h

just 

```
vmove usr/include/gpsim
```

is enough

File: srcpkgs/gpsim/template [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+license="GPL-2.0-or-later"
+homepage="http://gpsim.sourceforge.net/"
+distfiles="https://sourceforge.net/projects/gpsim/files/gpsim/${version}/gpsim-${version}.tar.gz"
+checksum=e1927312c37119bc26d6abf2c250072a279a9c764c49ae9d71b4ccebb8154f86
+
+gpsim-devel_package() {
+	short_desc+=" - development files"
+	depends="gpsim-${version}_${revision} gtk+-devel popt-devel readline-devel"
+	pkg_install() {
+		vmove "usr/include/gpsim"
```
Comment (maxice8) [inline srcpkgs/gpsim/template]: no need to be quoted because it is not globbing

**Comment by maxice8** [comment]
Please notify when proposed changes are made

**Comment by bobertlo** [comment]
@maxice8 ok fixed that

**Pull request merged**